core/src/it/scala/org/ensime/intg/DebugTest.scala

```diff
@@ -274,47 +488,50 @@ trait DebugTestUtils {
     expectMsg(TrueResponse)
 
     val vm = VMStarter(config, className)
```

Comment:
Yeah, I am as well. To be honest, I left it with the intention of removing that code since there is similar deployment code used in the debugger API under the `JDITools` object, but it needs to have a way for you to tie in to the standard output/error and redirect. The issue is #1456 with regard to that refactoring.